src/main/java/ca/uhn/fhir/jpa/starter/CustomAuthorizationInterceptor.java

```diff
@@ -29,6 +29,7 @@
 	private static final String APIKEY_HEADER = "x-api-key";
 	private static final String APIKEY = System.getenv("APIKEY");
```

Comment:
Please update the `docker-compose.yml` to include the env var changes - it still has the old env var names. It would be good for it to include defaults for the `APIKEY*` and `OAUTH*` settings.